apps/account-api/src/controllers/v2/accounts-v2.controller.ts

```diff
@@ -0,0 +1,67 @@
+import { SiwfV2Service } from '#account-api/services/siwfV2.service';
+import { SCHEMA_NAME_TO_ID } from '#types/constants/schemas';
+import { WalletV2RedirectRequestDto } from '#types/dtos/account/wallet.v2.redirect.request.dto';
+import { WalletV2RedirectResponseDto } from '#types/dtos/account/wallet.v2.redirect.response.dto';
+import { Controller, HttpCode, HttpStatus, Logger, HttpException, Query, Get } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+
+// # SIWF Wallet API V2
```

Comment:
This documentation is a copy of the docs at https://projectlibertylabs.github.io/siwf/v2/docs/ for easy reference while this is being built. Better docs coming once the entire controller is complete